tests/e2e/pact/specs/eps-auth.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { InteractionObject } from "@pact-foundation/pact"
+import * as jestpact from "jest-pact"
+import supertest from "supertest"
+
+jestpact.pactWith(
+  {
+    spec: 3,
+    consumer: `nhsd-apim-eps-test-client+${process.env.PACT_VERSION}`,
+    provider: `${process.env.PACT_PROVIDER}+${process.env.PACT_VERSION}`,
```

Comment:
This will overwrite the previous pact, effectively replacing all the model example tests with just the 3 invalid auth ones